source/validate.cpp

```diff
@@ -299,23 +299,42 @@ spv_result_t spvValidateBinary(const spv_const_context context,
   }
 
   // Create the ValidationState using the context.
-  ValidationState_t vstate(&hijack_context);
+  // This interface is used for default command line options.
+  ValidationState_t vstate(&hijack_context, nullptr);
```

Comment:
Isn't this dangerous?    What parts of the code assume a non-null options object?
Wouldn't it be better to create an options object that we destroy at the end of this scope?